emspayafterpay/emspayafterpay.php

```diff
@@ -244,7 +265,25 @@ protected function isSetShowForIpFilter()
         }
         return false;
     }
-    
+
+    /**
+     * check if the EMS_AFTERPAY_COUNTY_ACCESS is set,
+     * if so, only display if user is from that counties
+     *
+     * @return boolean
+     */
+    protected function CountryAccess($idusercountry)
+    {
+        $ems_afterpay_country_access = Configuration::get('EMS_AFTERPAY_COUNTRY_ACCESS');
+        if (strlen($ems_afterpay_country_access)) {
```

Comment:
Function 'strlen' return integer, but you use this like boolean.
It's bad usage of the function. You can use instead function 'empty()'  or if you want to use 'strlen'  formulate corretly statement like 'strlen(x) > y'